frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts

```diff
@@ -442,3 +443,15 @@ export const getCellsFromResults = (results: TektonResultsRun[]): ResultCells =>
     rows: results.map((result) => [result.name, result.value]),
   };
 };
+
+type GetMatchedPVCs = (
+  pvcResources: PersistentVolumeClaimKind[],
+  ownerReference: string,
+) => PersistentVolumeClaimKind[];
+
+export const getMatchedPVCs: GetMatchedPVCs = (pvcResources, ownerReference) => {
+  return pvcResources.filter((pvc) => {
+    const { ownerReferences } = pvc.metadata;
+    return ownerReferences.find((reference) => reference.name === ownerReference);
```

Comment:
`metadata.ownerReferences` is an optional field, If you call this function with a PVC without ownerReferences chances are this will throw a error in console, so add an empty array as a fallback value and you could also add test for the same.

Reply:
updated. checking Kind as well, doing a null check and added tests